packages/n-back/examples/index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <script src="https://unpkg.com/jspsych"></script>
+  <script src="https://unpkg.com/@jspsych/plugin-html-keyboard-response"></script>
+  <script src="../dist/index.global.js"></script>
+  <link rel="stylesheet" href="https://unpkg.com/jspsych/css/jspsych.css">
+</head>
+
+<body></body>
+<script>
+
+  const timeline = [];
+
+  const jsPsych = initJsPsych();
```

Comment:
it's usually good form to display the data at the end of a trial so that we can debug the data, you can just do it as such:
```js
const jsPsych = initJsPsych({
  on_finish: function() {
    jsPsych.data.displayData();
  }
});
```